src/main/java/seedu/address/logic/commands/tag/FindCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.commands.tag;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Lists all contacts or sale items associated with the specified tag.
+ */
+public class FindCommand extends Command {
+    public static final String COMMAND_WORD = "tag find";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Finds all the contacts or sales items associated with this tag. "
+            + "Note that all contacts or sales associated with this tag "
+            + "will be updated automatically with the updated tag.\n"
```

Comment:
Is this part relevant?

Reply:
Yep, this will be relevant once my next PR is in.